### ISSUE TITLE
Focus ring of links in header

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/earthworks#1257]
<img src="https://github.com/user-attachments/assets/4867b90e-9612-42eb-9ff2-cbd90e404228" width="300">

<img src="https://github.com/user-attachments/assets/493a6e76-7f4e-43a2-b183-ccd0e93988b7" width="400">

Focus ring on the links on the top right should be the default white/blue ring like it's shown for EarthWorks link that goes to home, otherwise it won't meet contrast requirements for accessibility.

**Comment by dnoneill** [comment]
Updated in component library https://github.com/sul-dlss/component-library/pull/31
